README.md

```diff
@@ -0,0 +1,33 @@
+# Pre-commit Hooks
+
+This repository contains [pre-commit](https://pre-commit.com/) hooks developed for Exasol projects.
+Each hook defines a check that pre-commit automatically installs and runs before each commit.
+
+## Available Hooks
+
+ * [SerialVersionUID change checker](serialVersionUIDChangeChecker/README.md)
+ 
+ Checkout the [pre-commit website](https://pre-commit.com/hooks.html) for a list of other pre-commit hooks.
+ 
+ ## Usage
+ 
+ You can define which hooks pre-commit should used for a specific project using a `.pre-commit-config.yaml` 
+ file in the projects root directory.
+ For more details see the [pre-commit quickstart guide](https://pre-commit.com/#quick-start).
+ This file is also versioned using the projects git.
```

Comment:
What does it mean?

Reply:
```suggestion
Usually you also commit this file into the projects git repository.
```

Reply:
Yes, it's better. Change it and then I approve